### PR TITLE
Prepare for Hubble v0.8.0 CLI release

### PR DESCRIPTION
**Pull request by gandro**
https://github.com/cilium/hubble/blob/master/RELEASE.md

### CODE REVIEW

**Review by rolinh — CHANGES_REQUESTED** [review]
Thanks! As we have made a couple of breaking changes, could we highlight them in the release notes? Maybe a section/text before the list of changes.

I think the following PRs contain breaking changes: #535 #533 #536 #542. We should at least give a brief warning/note for each of these.

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,135 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [v0.8.0] - 2021-05-03
+
+This release coincides with Cilium 1.10 and has support for a new API additions
```
Comment (rolinh) [inline CHANGELOG.md:11]:
```suggestion
This release coincides with Cilium 1.10 and has support for new API additions
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,135 @@ All notable changes to this project will be documented in this file.
...
+added in Cilium. Some of the new API features include support for agent and
+debug events (#537), as well as prelimary support for the experimental Hubble
+Recorder API (#530). Both of these features are currenlty only available via
+the local unix domain socket. Other API features include filtering by TCP flags
+(#461), IP version (#505) and node name (#412).
+
+Hubble CLI 0.8 also comes with improvements to the CLI utility, such as reading
+flows and filtering flows from stdin (#524), more flexible timestamp format
+printing (#509), support for Apple silicon (#488), as well as miscellaneous
+flag improvements and additions (#411, #420, #421, #443).
```
Comment (rolinh) [inline CHANGELOG.md:21]:
```suggestion
flag improvements and additions (#411, #420, #421, #443). It also contains a new `hubble list` subcommand which, when targeting Hubble Relay, lists all Hubble enabled nodes (#427).
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,135 @@ All notable changes to this project will be documented in this file.
...
+
+*Breaking Changes*
+
+In accordance with semver 0.x releases, this release contains a few
+breaking changes to the Hubble command-line interface:
+
+* The new default Hubble API endpoint (specified with `--server`) is now
+  `localhost:4245` to ease usage with Hubble Relay. To connect to the local
+  unix domain socket, use `--server unix:///var/run/cilium/hubble.sock` or set
+  the `HUBBLE_SERVER` environment variable. (#535)
```
Comment (rolinh) [inline CHANGELOG.md:31]:
```suggestion
  the `HUBBLE_SERVER` environment variable (default within a Cilium container). (#535)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -6,6 +6,135 @@ All notable changes to this project will be documented in this file.
...
+* The new default output format is now always `compact` regardless of the. To
```
Comment (rolinh) [inline CHANGELOG.md:32]:
```suggestion
* The new default output format is now always `compact` regardless of being in follow-mode or not. To
```

**Review by rolinh — COMMENTED** [review]
Awesome :rocket: ! I have a few amendments/suggestions, see below.

File: CHANGELOG.md (hunk 4 above)
Comment (gandro) [reply CHANGELOG.md:32]: Oops :grimacing: Thanks!

**Review by rolinh — APPROVED** [review]
:rocket:

**Comment by gandro** [comment]
This was not supposed to target `master`. Not a big problem though, we'll just have to forward v0.8 manually.

**Pull request merged**